core/object/object.cpp

```diff
@@ -1248,6 +1262,9 @@ void Object::_add_user_signal(const String &p_name, const Array &p_args) {
 	}
 
 	add_user_signal(mi);
+
+	if (signal_map.has(p_name))
+		signal_map.getptr(p_name)->removable = true;
```

Comment:
I think we try to keep the `{}` on all if but I'm not 100% sure
```suggestion
	if (signal_map.has(p_name)) {
		signal_map.getptr(p_name)->removable = true;
	}
```